sklearn/svm/_classes.py

```diff
@@ -690,6 +691,10 @@ class SVC(BaseSVC):
         5-fold cross-validation, and `predict_proba` may be inconsistent with
         `predict`. Read more in the :ref:`User Guide <scores_probabilities>`.
 
+        ..deprecated:: version 1.8 parameter probability will be deprecated in future
+        versions, use CalibratedClassifierCV(SVC(), ensemble=False) instead
+        of SVC(probability=True)
```

Comment:
```suggestion
        ..deprecated:: 1.8
          Parameter `probability` will be removed in 1.10.
          Use `CalibratedClassifierCV(SVC(), ensemble=False)` instead of
          `SVC(probability=True)`.
```